clash-lib/src/Clash/Primitives/Verification.hs

```diff
@@ -0,0 +1,141 @@
+{-# LANGUAGE RankNTypes            #-}
+{-# LANGUAGE ScopedTypeVariables   #-}
+{-# LANGUAGE TemplateHaskell       #-}
+{-# LANGUAGE OverloadedStrings     #-}
+{-# LANGUAGE ViewPatterns          #-}
+
+module Clash.Primitives.Verification (assertBBF, hideCvResultTF) where
+
+import Data.Either
+
+
+import qualified Control.Lens                    as Lens
+import           Control.Monad.State             (State)
+import           Data.Text.Prettyprint.Doc.Extra (Doc)
+import qualified Data.Text                       as Text
+import           Data.Semigroup.Monad            (getMon)
+import           GHC.Stack                       (HasCallStack)
+
+import           Clash.Annotations.Primitive     (HDL(..))
+import qualified Clash.Backend
+import           Clash.Backend
+  (Backend, blockDecl, hdlKind, expr)
+import           Clash.Core.Term                 (Term(Var))
+import           Clash.Core.TermLiteral
+  (uncheckedTermToData, termToDataError)
+import           Clash.Core.Util                 (termType)
+import           Clash.Util                      (indexNote)
+import           Clash.Netlist                   (mkExpr)
+import           Clash.Netlist.Util              (stripVoid)
+import qualified Clash.Netlist.Util
+import           Clash.Netlist.Util              (id2identifier)
+import           Clash.Netlist.Id                (IdType(Basic))
+import           Clash.Netlist.Types
```

Comment:
This file is all new, and yet does not follow our newly accepted "let's not add all these extra spaces for alignment" approach to imports

Reply:
Did we settle on that for imports? I'm not sure.

Reply:
Although we probably should. I really don't like aligning imports.

Reply:
We didn't make any changes, but I thought we reached consensus on avoiding unnecessary alignment in new things (given we take the same approach to things like equals signs and arrows in case expressions)

Reply:
Ok, I'll bring it up. I think it's a good idea to explicitly discuss this.

Reply:
I've brought it up, I'll add the results of the discussion to the style guide.